connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectDistributed.java

```diff
@@ -27,12 +27,7 @@
 import org.apache.kafka.connect.runtime.distributed.DistributedHerder;
 import org.apache.kafka.connect.runtime.isolation.Plugins;
 import org.apache.kafka.connect.runtime.rest.RestServer;
-import org.apache.kafka.connect.storage.ConfigBackingStore;
-import org.apache.kafka.connect.storage.Converter;
-import org.apache.kafka.connect.storage.KafkaConfigBackingStore;
-import org.apache.kafka.connect.storage.KafkaOffsetBackingStore;
-import org.apache.kafka.connect.storage.KafkaStatusBackingStore;
-import org.apache.kafka.connect.storage.StatusBackingStore;
+import org.apache.kafka.connect.storage.*;
```

Comment:
Considering we're only using a few classes from this package, should we just keep the individual imports instead of using star here?